media-libs/libgphoto2/files/libgphoto2-2.5.30-c99-fix-prototype.patch

```diff
@@ -0,0 +1,11 @@
+--- libgphoto2-2.5.30.orig/tests/test-pedantic-compilation.c	2024-03-28 06:50:08.823424429 +0000
```

Comment:
If it's a backport, please include a link to the commit it's from (ideally with the full metadata from e.g. gh's URL with .patch appended, you can copy/paste the headers).

Reply:
It's not a backport. I need to push the patch upstream, yes.

Reply:
https://github.com/gentoo/gentoo/pull/35960/commits/4985b4bc8b179062800efbbc1ef9f41a68990439 said "backport" but I maybe misunderstood it - you meant backport to an older version in Gentoo, or what?

Please do submit it upstream and include a link in the patch.